proto/buf.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/datachainlab/ibc-hd-signer
+deps:
+  - buf.build/cosmos/gogo-proto
+  - buf.build/cosmos/ibc
```

Comment:
Maybe this repository only depends gogo-proto, and not ibc?

Reply:
fixed